edge-functions/serverside-analytics/pages/_middleware.ts

```diff
@@ -0,0 +1,41 @@
+import type { NextFetchEvent, NextRequest } from 'next/server'
+import { NextResponse } from 'next/server'
+
+const PUBLIC_FILE = /\.(.*)$/
+
+async function logPageView(req: NextRequest) {
+  // ignore static assets from being tracked,
+  if (
+    // process.env.NODE_ENV !== 'production' || // uncomment this line to track only production requests
```

Comment:
I would prefer not to have this check but instead have the analytics tool (the fetch below) do something different for development, like doing a console.log instead.